Frontend/src/app/auth/pages/register/register.component.html

```diff
@@ -60,7 +60,16 @@ <h4 class="text-black">Register</h4>
           <p class="error" *ngIf="error">{{ error }}</p>
           
           <div class="form-group">
-            <button nbButton fullWidth status="success" (click)="register(registerForm.value)">Register</button>
+            <button 
+              nbButton 
+              fullWidth 
+              status="success" 
+              (click)="register(registerForm.value)"
+              [nbSpinner]="isLoading"
+              [disabled]="isLoading"
+              nbSpinnerStatus="danger">
```

Comment:
I feel like green would look better here than red, what do you think? Red is a scary color.